### PR TITLE
Lag mitigation for APS (heavy only)

### PR DESCRIPTION
**Pull request by spooker0**
This is a more foolproof method of addressing bypassing of APS under heavy lag conditions.

### CODE REVIEW

File: scripts/DAPS/Scripts/APS/fn_LagMitigation.sqf [hunk 1]
```diff
@@ -0,0 +1,62 @@
+params ["_unit", "_selection", "_damage", "_source", "_projectile", "_hitIndex", "_instigator", "_hitPoint", "_directHit"];
+
+if (!alive _unit) exitWith {	// vehicle already dead?
+	nil;
+};
+if (!(_unit call DAPS_fnc_active)) exitWith {	// APS active and has charges?
+	nil;
+};
+if (_unit distance _source < 40) exitWith {	// too close?
```
Comment (MrThomasM) [inline scripts/DAPS/Scripts/APS/fn_LagMitigation.sqf]: change this to 30 meters so it matches the other ranges

**Comment by MrThomasM** [comment]
I will have a look at this tmrw morning and check the rest.

**Comment by MrThomasM** [comment]
of the code*

**Review by MrThomasM — CHANGES_REQUESTED** [review]

File: scripts/DAPS/Scripts/APS/fn_LagMitigation.sqf (hunk 1 above)
Comment (spooker0) [reply scripts/DAPS/Scripts/APS/fn_LagMitigation.sqf]: Done.

**Pull request merged**